.github/ISSUE_TEMPLATE/1-lts-release-checklist.md

```diff
@@ -0,0 +1,71 @@
+---
+name: "🥇 New LTS release checklist"
+labels: lts
+about: Track work required for a new LTS release
+---
+
+# Next LTS release
+
+## Release Lead
+
+<!-- 
+The release lead is the person who makes sure that all steps are completed
+Not necessarily the person doing all the work
+
+This role should rotate between LTS releases
+-->
+
+@<github-username of release lead>
+
+## Prep work
+
+- [ ] LTS baseline selected
```

Comment:
Would it makes sense to have a link to the discussion happening in the ML?

Reply:
Yes this is missing the pre selection step of looking for consensus on a baseline